rel-eng/releasers.conf

```diff
@@ -3,6 +3,10 @@ releaser = tito.release.KojiReleaser
 autobuild_tags = katello-nightly-rhel5 katello-nightly-rhel6 katello-nightly-rhel7 katello-nightly-fedora25 katello-nightly-fedora24 katello-thirdparty-rhel7
 builder.rpmbuild_options = --define "foremandist .fm1_16"
 
+[koji-katello-client]
+releaser = tito.release.KojiReleaser
+autobuild_tags = katello-nightly-rhel5 katello-nightly-rhel6 katello-nightly-rhel7 katello-nightly-fedora25 katello-nightly-fedora24 katello-thirdparty-rhel7
```

Comment:
Want to trim the autobuild_tags for the `koji-katello` releaser now that we have a client focused one?